kroxylicious/pom.xml

```diff
@@ -39,6 +39,11 @@
             <version>${kroxyliciousApi.version}</version>
         </dependency>
         <!-- We depend on the specific Netty dependencies not netty-all to reduce the size of fatjars -->
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
I'd rather we maintain our own Preconditions class than add a dependency for those two methods. WDYT?

Reply:
Agreed, I'll eliminate it.